### ISSUE TITLE
Missing alt_condition attribute from weather.weatherbit sensor

### ISSUE THREAD

**Issue by Landshark77** [briis/weatherbit#47]
Running 1.0.1.

Is it possible to add back the alt_condition attribute to the weather.weatherbit sensor?  Or did it move somewhere else?

I leveraged this value to determine the background image to display.  For example, if the alt_condition was "partlycloudy-night", I display a partly cloudy night image, etc....

**Comment by briis** [comment]
I left it out on purpose, but I can surely add it back. I was leaving out non standard stuff, to see if anyone missed them - as you do with this one.
Will get it done for the next release.

**Comment by Mariusthvdb** [comment]
NIce! @Landshark77 would you care to share that config please?

using 
```
type: custom:dark-sky-weather-card
card_mod:
  style: |
    ha-card {
      background: url("/local/images/weather-background-{{states('sun.sun')}}.png");
      background-size: 100% 400px;
    }
```
myself on the Darksky card, this the way you do it too?

**Comment by briis** [comment]
Just released 1.0.2 where this is now fixed.
